release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+For more information, see xref:../operating/manage_security_policies/security-policy-reference.adoc#high-sev-security-policies_security-policy-reference[High severity security policies].
+
+//ROX-26282
+[id="automatic-baseline-locking_{context}"]
+=== Ability to automatically lock baselines to improve process execution policy
+
+This new feature is a major improvement to our process execution policy. It directly addresses customer challenges by automating the process of locking baselines, which was previously a time-consuming and manual task for each deployment. This frees up security teams to focus on more critical work.
+Furthermore, this change allows for a more proactive security approach. Instead of waiting for a deployment to exist before setting up an alert, you can now define a policy for a specific scope, such as a namespace. Any new deployment in that scope will automatically be raise alerts, ensuring consistent security from day one.
+
+For more information, see xref:../operating/evaluate-security-risks.adoc#auto-lock-process-baselines_evaluate-security-risks[Configuring auto-lock for process baselines].
+
+//ROX-26289
+[id="vuln-reporting-enhancements_{context}"]
+=== Vulnerability reporting enhancements
+
+With this release, you can now significantly enhance vulnerability management workflows by directly exporting CSV files from the vulnerability management pages. This feature empowers you to use granular filters and create on-demand, view-based reports and provides flexibility in analyzing data and addressing specific security concerns.
+
+Additionally, you can now generate view-based reports directly from individual image and deployment detail pages.
+
+For more information about creating and downloading vulnerability reports, see xref:../operating/manage-vulnerabilities/vulnerability-reporting.adoc#vulnerability-reporting[Vulnerability reporting].
+
+//ROX-28029
+[id="scannerv4-sbom-delegated-scans_{context}"]
+=== SBOM creation is generally available and supports delegated scanning
+
+With this release, using {product-title-short} to generate a Software Bill of Materials (SBOM) from scanned container images is generally available (GA). SBOM generation includes images scanned with the {product-title-short} delegated scanning feature. These SBOMs provide a detailed overview of all software components, dependencies, and libraries within an application. The SBOMs created by {product-title-short} are of the link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf["Analyzed"] type and conform to the SPDX 2.3 specification.
```

Comment:
IBM Style > Language and grammar > Abbreviations > Spelled out forms
capitalization of "Software Bill of Materials"

Why are the words in Software Bill of Materials capitalized--is this an OpenShift rule?

Reply:
I don't think so - I see it both ways on other Red Hat docs. Previously, we've used caps so I followed that for consistency.